### ISSUE TITLE
(Crispy Heretic) Total Timer in Intermission Screen Will Reset After Loading a Game

### ISSUE THREAD

**Issue by GarrettChan0** [fabiangreffrath/crispy-doom#673]
### Background

Version of Crispy Doom: Crispy Heretic 5.9.2

Operating System and version: Windows 7

### Bug description

Observed behavior: If you load a game, the total timer in the intermission screen will be reset and only have the time for the current level.

Example:

I took 0:30 on Map01 and I took 0:25 on Map02. The expected intermission screen after Map02 should say:

time :25
total :55

but instead, if you loaded a game during any time in this session, it will say:

time :25
total :25

**Comment by fabiangreffrath** [comment]
Yes, that's because the accumulated time isn't (yet) stored in savegames. See https://github.com/fabiangreffrath/crispy-doom/issues/648
